python/tk_desktop/desktop_window.py

```diff
@@ -1188,7 +1220,8 @@ def __launch_app_proxy_for_project(self, project, requested_pipeline_configurati
 
             # If no pipeline configurations were found in Shotgun, show the
             # 'Advanced project setup...' menu item.
-            if not pipeline_configurations:
+            # If there is no pipeline configuration set for the current project, add the Advanced Project Setup menu.
+            if not any(True if pc["project"] else False for pc in pipeline_configurations):
```

Comment:
If a site level pipeline configuration was present, we wouldn't have been able to run `Advanced Project Setup`.